pkg/cmd/server/start/start_master.go

```diff
@@ -558,6 +558,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunEndpointController()
 		kc.RunNamespaceController()
 		kc.RunPersistentVolumeClaimBinder()
+		kc.RunPersistentVolumeProvisioner(pvKClient)
```

Comment:
I'd like to not add an account at this point.

The next iteration of provisioning (which includes recycling operations) contains a "service account name" bit, which will be the account to use for that specific plugin.  Users will be able to mix and match according to the needs of the plugin.